jpm/cgen.janet

```diff
@@ -403,8 +403,12 @@
       (set is-first false)
       (emit-type (arg 1))
       (prin " " (arg 0)))
-    (prin ") ")
-    (emit-do body))
+    (prin ")")
+    (if (empty? body)
+      (print ";")
+      (do
+        (print " ")
```

Comment:
I think this is supposed to be `(prin " ")`, otherwise you are just adding trailing whitespace

Reply:
Indeed. I'm AFK. I'll get to it later. Thanks